ui/libproject/devstart.rb

```diff
@@ -89,6 +96,12 @@ def clean_git_hooks()
 })
 
 Common.register_command({
+  :invocation => "rebuild-image",
```

Comment:
The README says to run `./project.rb` to see all available commands. I'm wary of documenting each one because it's so easy to forget to add/remove stuff.